src/Sulu/Bundle/ContentBundle/Twig/ContentTwigExtension.php

```diff
@@ -138,7 +138,10 @@ public function needsAddButtonFunction(PropertyInterface $property)
      */
     public function isMultipleTest($property)
     {
-        return $property->getMinOccurs() > 1;
+        return
+            !is_null($property->getMaxOccurs()) ||
+            $property->getMinOccurs() > 1 ||
+            $property->getMinOccurs() < $property->getMaxOccurs();
```

Comment:
But will accomplish the feature to have 0 or 1 element (showing the button to add the element, without any empty fields below)